kbase-extension/scss/partials/_narrative.scss

```diff
@@ -270,41 +251,48 @@
     .select2-selection__placeholder {
       color: rgb(223 0 2);
     }
+
+    /* style for Import As dropdown container arrow */
+    .select2-selection__arrow {
+      position: absolute;
+      top: 30.95%;
+      right: 5.44%;
+      bottom: 30.95%;
+      left: 87.76%;
+      height: 16px;
+
+      b {
+        border-color: rgb(223 0 2) transparent transparent;
+      }
+    }
+
+    &.kb-staging-table-body__import-type-selected .select2-selection__arrow b,
+    &:focus .select2-selection__arrow b {
+      border-color: rgb(136 136 136) transparent transparent;
+    }
   }
 
   .select2-container--open .select2-selection__placeholder {
     color: rgb(0 0 0);
   }
 }
 
-/* Import as dropdown placeholder text is red */
+/* TODO: can this class be merged with the one above? */
 
-/* Change placeholder color when user clicks in */
+/* When user clicks into the dropdown or chooses a selection box change border color */
 
-.select2-container--focus .select2-selection__placeholder {
-  color: rgb(0 0 0);
+.kb-staging-table-body__import-dropdown.kb-staging-table-body__import-type-selected,
+.select2-container--focus .kb-staging-table-body__import-dropdown,
+.select2-container--open .kb-staging-table-body__import-dropdown {
+  border: 1px solid rgb(196 196 196);
 }
 
-/* style for Import As dropdown container arrow */
-
-.kb-staging-table-body .kb-staging-table-body__import-dropdown {
-  .select2-selection__arrow {
-    position: absolute;
-    top: 30.95%;
-    right: 5.44%;
-    bottom: 30.95%;
-    left: 87.76%;
-    height: 16px;
+/* Import as dropdown placeholder text is red */
 
-    b {
-      border-color: rgb(223 0 2) transparent transparent;
-    }
-  }
+/* Change placeholder color when user clicks in */
 
-  &.kb-staging-table-body__import-type-selected .select2-selection__arrow b,
-  &:focus .select2-selection__arrow b {
-    border-color: rgb(136 136 136) transparent transparent;
-  }
+.select2-container--focus .select2-selection__placeholder {
```

Comment:
Should this be like this, or should it be nested under `.kb-staging-table-*`?

Reply:
i think it makes sense nested under `.kb-staging-table`, we don't need the extra class declarations for it to apply, but from a comprehension and maintenance standpoint it fits well under there (so it's clear that the placeholder color changes between resting state and focus)